routers/products.mjs

```diff
@@ -0,0 +1,29 @@
+import { sendJsonData } from '../utils/response';
+
+export function setProductsApi(router) {
+  router.route('/products')
+    .get(getProductListTreatment)
+    .post(createProductTreatment);
+
+  router.route('/products/:id')
+    .get(getProductTreatment);
+
+  router.route('/products/:id/reviews')
+    .get(getProductReviewsTreatment);
+}
+
+export function getProductListTreatment(req, res) {

```

Comment:
You can move such kind of methods into appropriate files of _controller_ folder.
Let's say a controller takes care of destination (it might be file, database, variable and etc.) to pull data from. And outside of it no one should be worried about that and use methods the controller provides to get data